run_dir/static/js/projects.js

```diff
@@ -843,21 +843,52 @@ $("#allFields").change(function(e){
   updateTableFields("");
 })
 
+$('#reorganiseReorderbtn').on("click", function() {
+  console.log('hi');
+  $("#tHeaderListul").empty();
+  tHList="";
+  $("#allColFields input[class='filterCheckbox']:checked").each(function(i, elem){
+     tHList+='<li data-name="'+$(this).prop('name')+'">'+$(this).data('displayname')+'</li>';
+   })
+  $('#tHeaderListul').append(tHList);
```

Comment:
The part above of here could potentially be in a function since it is very similar to the code used to initialise the `#tHeaderListul` in the first place, right? Not mandatory though.

Reply:
Done!